FluentUI/src/main/java/com/microsoft/fluentui/persistentbottomsheet/sheetItem/PersistentBottomSheetAdapter.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.fluentui.persistentbottomsheet.sheetItem
+
+import android.content.Context
+import android.support.annotation.StyleRes
+import android.support.v7.widget.RecyclerView
+import android.view.ContextThemeWrapper
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import com.microsoft.fluentui.R
+import com.microsoft.fluentui.bottomsheet.BottomSheetItem
+import com.microsoft.fluentui.bottomsheet.BottomSheetItem.Companion.NO_ID
+import com.microsoft.fluentui.bottomsheet.getImageTint
+import com.microsoft.fluentui.listitem.ListItemView
+import com.microsoft.fluentui.util.createImageView
+
+class PersistentBottomSheetAdapter(private val context: Context, private val items: List<BottomSheetItem>, @StyleRes private val themeId: Int) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```

Comment:
Why do we need this? If we have bottomsheetadapter

Reply:
removing